remote/remote_state_s3.go

```diff
@@ -435,9 +420,19 @@ func CreateS3BucketWithVersioningSSEncryptionAndAccessLogging(s3Client *s3.S3, c
 	}
 
 	if config.SkipBucketAccessLogging {
-		terragruntOptions.Logger.Printf("Access Logging is disabled for the remote state AWS S3 bucket %s using 'skip_bucket_accesslogging' config.", config.remoteStateConfigS3.Bucket)
-	} else if err := EnableAccessLoggingForS3BucketWide(s3Client, &config.remoteStateConfigS3, terragruntOptions); err != nil {
-		return err
+		terragruntOptions.Logger.Printf("___WARNING___: The terragrunt configuration option 'skip_bucket_accesslogging' is now deprecated. To enable access logging for bucket %s, please provide property `accesslogging_bucket_name` in the terragrunt config file. For more details, please refer to the Terragrunt documentation.", config.remoteStateConfigS3.Bucket)
+	}
+
+	if config.AccessLoggingBucketName != "" {
+		if err := CreateS3Bucket(s3Client, aws.String(config.AccessLoggingBucketName), terragruntOptions); err != nil {
+			terragruntOptions.Logger.Printf("Access Logging is disabled for the remote state AWS S3 bucket for bucket %s", config.remoteStateConfigS3.Bucket)
```

Comment:
This message needs to be updated.

Reply:
Hm, I'm unsure what you meant here

Reply:
ahh. ignore the above ☝️ I _think_ I understand your train of thought: the error message is about the creation of the bucket, not the access logging 👍 Let me know if my understadning is correct.

Reply:
Yup that is correct. E.g., I would switch this to `("Error creating access logging bucket: %s", err)`.